lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -5039,7 +5043,8 @@ void TypeChecker::checkConformancesInContext(DeclContext *dc,
     if (auto classDecl = dc->getSelfClassDecl()) {
       if (Context.LangOpts.EnableObjCInterop &&
           isNSCoding(conformance->getProtocol()) &&
-          !classDecl->isGenericContext()) {
+          !classDecl->isGenericContext() &&
+          !classDecl->hasClangNode()) {
```

Comment:
When does this come up?

Reply:
Retroactive conformance of imported class to NSCoding. We already checked for hasClangNode() before, but I moved it.

Reply:
\*shrug\* All right. It's true that those won't have unstable names either.